Makefile

```diff
@@ -19,7 +19,9 @@ dryRelease:
 # remember to remove the -SNAPSHOT suffix from the version
 # promotes the release to maven central
 doRelease:
-	./gradlew publish --no-daemon
+	cd scripts
+	kotlinc -script release.kts -- -d ../distributions | sh
+	cd ..
 	./gradlew closeAndReleaseRepository
```

Comment:
mvnw does not close and promote, so using the gradle's plugin task that does it otherwise we have to click buttons